Dockerfile

```diff
@@ -1,21 +1,39 @@
-FROM node:10.13.0
+FROM node:10.13-alpine
 
-# The key bits here are making sure we install, libsecret-1-dev libglib2.0-dev
-RUN apt-get update -qq && apt-get install -y \
-  libsecret-1-dev libglib2.0-dev && \
+RUN apk add bash curl git \
+  make gcc g++ python \
+  libsecret-dev glib-dev && \
   rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Don't need this line as this was removing apt's caches - can add the flag `--no-cache` to `apk add` above to avoid the caches for apk